tendrl-common.spec

```diff
@@ -7,13 +7,15 @@ Source0: %{name}-%{version}.tar.gz
 License: LGPLv2+
 URL: https://github.com/Tendrl/common
 
+BuildRequires: ansible
```

Comment:
I remember on rhel by default ansible 2.2 was not getting downloaded. We should explicitly mention > 2.2 here if you use any ansible 2.2 specific feature.